File: ao/simple/kuhn/KuhnPlayer.java
package ao.simple.kuhn;

import ao.simple.kuhn.state.KuhnState;

/**
 *
 */
public interface KuhnPlayer
{
    public KuhnAction
            act(KuhnCard  hole,
                KuhnState state);

//    public void handEnded();
}

File: trunk/ao-ai-cfr/trunk/src/main/scala/ao/learn/mst/kuhn/play/KuhnPlayer.scala
package ao.learn.mst.kuhn.play

import ao.learn.mst.kuhn.action.KuhnAction
import KuhnAction._
import ao.learn.mst.kuhn.state.KuhnState


//----------------------------------------------------------------------------------------------------------------------
trait KuhnPlayer
{
  //--------------------------------------------------------------------------------------------------------------------
  def act(state : KuhnState) : KuhnAction
}